faq/cohort-subs-guide.rst

```diff
@@ -0,0 +1,149 @@
+
+Top-level Sample records can be created in BioSamples using the `BioSamples uploader tool <https://www.ebi.ac.uk/biosamples/docs/cookbook/upload_files>`_.
+
+.. note ::
+    The ENA and the `EGA (European Genome Phenome Archive) <https://ega-archive.org/>`_ are the only archives which integrate
+    BioSample records into their :doc:`database structure <submit/general-guide/metadata>`. For data deposited at other
+    archives, additional BioSample records may need to be created (in BioSamples) to represent those data.
+
+2. Submit Pathogen Sequence data to the ENA
+'''''''''''''''''''''''''''''''''''''''''''
+
+The next step is to submit your pathogen nucleotide records (raw reads or assembly data) to the ENA.
+The :doc:`Pathogen Submissions Guide <faq/pathogen-subs-guide>` provides a quick introduction to the ENA and tips for
+Pathogen data submissions.
+
+
+2. Create the child Samples in additional database resources
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+The next step is to create your datasets in the correct database for the data type. The `EBI submissions wizard
+<https://www.ebi.ac.uk/submission/>`_ can help direct you to a resource to deposit your data.
+We can reccommend the following database resources for common data types:
+
+- For sensitive human nucleotide records and human clinical epidemiological data which requires controlled access, please
+  contact the `EGA (European Genome Phenome Archive) <https://ega-archive.org/>`_ to start a submission.
+- For expression data, or uncategorsied datasets, please use `ArrayExpress/BioStudies <https://www.ebi.ac.uk/biostudies/arrayexpress>`_
+
+After the datasets have been submitted in the appropriate databases, the required child Samples for linking can be created.
+The child samples will represent their relationship to the top-level Sample. Different samples can be used for different
+data types **and** for different time points. Please contact us if you have any doubts about setting up your sample structure.
+
+
+3. Link together the samples using BioSamples
+''''''''''''''''''''''''''''''''''''''''''''''
+
+Link your samples created from other EBI resources to the top-level sample using the ‘derived from’ curation
+on biosamples: https://www.ebi.ac.uk/biosamples/docs/references/api/submit#_submit_curation_object
+
+Link your samples created from other EBI resources to the top-level sample using the ‘derived from’ curation on
+BioSamples. The derived from relationship is used as follows, where the Source is the child Sample, and the Target is
+the top-level Sample:
+
+**Source sample** - *derived from* - **Target sample**
+
+**Child sample accession** - *derived from* - **Parent sample accession**
+
+For example, in the first linked dataset, the `EMC study <https://www.ebi.ac.uk/about/news/updates-from-data-resources/pathogens-portal-linked-dataset/>`_,
+the BioSamples relationship is as follows:
```

Comment:
I think it would also be helpful to include a link here to the IDTK page, as it contains more diagrams: https://www.infectious-diseases-toolkit.org/showcase/linked-cohort-data

Reply:
IDTK added with link title '[Erasmus Medical Cemter (EMC) study'